compute_tools/src/compute.rs

```diff
@@ -532,6 +532,50 @@ impl ComputeNode {
         Ok(())
     }
 
+    /// Start and stop a postgres process to warm up the VM for startup.
+    pub fn prewarm_postgres(&self) -> Result<()> {
+        info!("prewarming");
+
+        // Create pgdata
+        let pgdata = "/home/bojan/tmp/prewarm_pgdata"; // TODO fix this
+        create_pgdata(pgdata)?;
+
+        // Run initdb to completion
+        info!("running initdb");
+        let initdb_bin = Path::new(&self.pgbin).parent().unwrap().join("initdb");
+        Command::new(initdb_bin)
+            .args(["-D", pgdata])
+            .output()
+            .expect("cannot start initdb process");
+
+        // Write conf
+        use std::io::Write;
+        let conf_path = Path::new(pgdata).join("postgresql.conf");
+        let mut file = std::fs::File::create(conf_path)?;
+        writeln!(file, "shared_buffers=100MB")?; // TODO is this the default?
```

Comment:
https://www.postgresql.org/docs/current/runtime-config-resource.html

128 MB

Reply:
I mean what do we use? I've heard depends on pod vs vm, and sometimes it's half the available ram

Reply:
I see `16384` on my 0.25 vm and `65536` on my 0.25 pod. I'll use `65536`. Good enough for the experiment